aws/iam_types.go

```diff
@@ -0,0 +1,36 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// IAMUsers - represents all IAMUsers on the AWS Account
+type IAMUsers struct {
+	UserNames []string
+}
+
+// ResourceName - the simple name of the aws resource
+func (u IAMUsers) ResourceName() string {
+	return "iam"
+}
+
+// ResourceIdentifiers - The IAM UserNames
+func (u IAMUsers) ResourceIdentifiers() []string {
+	return u.UserNames
+}
+
+// Tentative batch size to ensure AWS doesn't throttle
+func (u IAMUsers) MaxBatchSize() int {
+	return 200
+}
+
+// Nuke - nuke 'em all!!!
+func (u IAMUsers) Nuke(session *session.Session, identifiers []string) error {
```

Comment:
could we name this explitily as `users`

Reply:
same applies to the function params above too!

Reply:
> NIT: could we name this explitily as `users`

You mean the `IAMUsers` struct?
(And then change its usage to `users` everywhere in the code/config?)

Reply:
> same applies to the function params above too!

Yeap! That makes a lot of sense... doing it now :smile:

Reply:
Yep, that's exactly what I meant - apologies for not being clearer to start with!